spec/persister/test_builder/cloud_manager.rb

```diff
@@ -0,0 +1,205 @@
+require_relative "../test_builder"
+
+class TestBuilder::CloudManager < TestBuilder
```

Comment:
we already have builder like this is specs, why do we add another?

Reply:
1) this is better place than in ManageIQ namespace
2) I didn't figure out how to include them to these specs, I spent many hours with writing correct "requires" but every time there was some `unitialized constant`